fuglu/src/fuglu/core.py

```diff
@@ -608,7 +608,7 @@ def startup(self):
             sys.stderr.write(
                 "Some plugins failed to load, please check the logs. Aborting.\n")
             self.logger.info('Fuglu shut down after fatal error condition')
-            sys.exit(1)
+            return 66
```

Comment:
that `66` looks arbitrary. maybe create a global  variable name for it so it is clear what it represents? ( something like `EXIT_FATAL=66` .... `return EXIT_FATAL` )

Reply:
Agree